src/utils/server-utils.ts

```diff
@@ -420,12 +432,20 @@ export async function getLayersAvailableDates(): Promise<AvailableDates> {
     (layer): layer is PointDataLayerProps => layer.type === 'point_data',
   );
 
+  const adminWithDateLayers = Object.values(LayerDefinitions).filter(
```

Comment:
Maybe this could be merged with pointDataLayers to avoid looping over the same data twice?